feat: Add AC operation modes and fade effect to climate template

Emericaa opened the pull request:
## Changes Made:

- Added support for AC operation modes in the climate template
- Implemented graph fill with fade effect for visual enhancement

Emericaa committed:
Update hc_climate_card.yaml

markwwd commented:
Corrected the row-gap here:
```
            grid:
              - grid-template-areas: |
                  'heat'
                  'cool'
                  'fan'
              - row-gap: **0.3rem**
```

The mode icons are otherwise misaligned.

Emericaa committed:
Update hc_climate_card.yaml

Variables correction:

variables:
    show_window_state:
    window_open_boolean:
    show_mode_state:
    show_graph:
    graph_entity:
    graph_color:

Emericaa committed:
Merge branch 'damianeickhoff:main' into main

Emericaa committed:
Mobile + Tablet

Mobile 100%
Tablet 1%

Emericaa committed:
Merge branch 'main' of https://github.com/Emericaa/HaCasa

damianeickhoff commented:
Hi @Emericaa 

Can you elaborate more on all the changes? I see a lot of icons and mobile + tablet. Also missing some images. 
Maybe you can add those?
